### PR TITLE
Add Automation is Hard by Johan Abildskov

### PR DESCRIPTION
**Pull request by RandomSort**
Hi! 👋 
I would love to give the talk Automation is Hard.

After coordinating with @kvarak, I have also graciously allowed remote talks during COVID-19. It is in a separate commit, so feel free to reject that.

Cheers!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ By adding your checkbox, you are approving [DevOps Malmö Meetup group guideline
 - [x] *2018-04-09* - [Learn Ansible automation with Red Hat](https://goo.gl/ThoJBB) by Magnus Glatz
 - [x] *2018-02-20* - [DevOps Skåne - introduction for students](https://goo.gl/K5sbk7) by Karen Har-yan, Fredrik Stål and Albert Rigo
 - [x] *2018-01-15* - [Basic Kubernetes](https://goo.gl/GTHwyi) by Karen Har-yan
+- [x] *2021-03-09* - Automation is Hard and we are doing it wrong
```
Comment (kvarak) [inline README.md]: Please add this line to the top ;)

**Review by gangsta — APPROVED** [review]
Looking forward!

File: README.md (hunk 1 above)
Comment (RandomSort) [reply README.md]: I mangled my history to make this change.

File: README.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ No date is currently known, we will update Readme once we know more.
 
 By adding your checkbox, you are approving [DevOps Malmö Meetup group guidelines for speakers](guidelines.md).
 
+- [x] *2021-03-09* - Automation is Hard and we are doing it wrong
```
Comment (kvarak) [inline README.md:18]: Add a link to talks/johan-abildskov.md?

Comment (gangsta) [inline README.md:18]: you could use following
[Johan Abildskov](talks/ talks/johan-abildskov.md)

**Pull request merged**